src/main/kotlin/racingcar/controller/GameController.kt

```diff
@@ -0,0 +1,25 @@
+package racingcar.controller
+
+import racingcar.domain.Cars
+import racingcar.domain.GameRound
+import racingcar.domain.RacingGame
+import racingcar.util.RandomGenerator
+import racingcar.view.GameWinnerResultViewModel
+import racingcar.view.RacingGameInput
+import racingcar.view.ResultView
+
+class GameController {
+    fun run(racingGameInput: RacingGameInput) {
+        val cars = Cars.createCars(racingGameInput.carNames)
+        val gameRounds = GameRound(racingGameInput.playCount)
+        val racingGame = RacingGame(cars, gameRounds, RandomGenerator())
+        val resultView = ResultView()
+
+        resultView.resolveTitle()
+
+        racingGame.play()
+
+        resultView.resolveCarsInfo(racingGame.carHistories)
+        resultView.resolveGameWinner(GameWinnerResultViewModel(racingGame.getWinnerInfo()))
```

Comment:
```suggestion
        
        val gameResult = racingGame.play()
        val winners = gameResult.findWinner()
        ...
```

와 같이, 객체를 리턴함으로써 메세지를 전달하는 방법도 참고해봐도 좋을거 같아요

Reply:
게임 마지막의 상태를 CarHistory로 반환하도록 수정해봤습니다! 덕분에 RacingGame의 책임이 가벼워진 것 같습니다 👍